packages/control/ocpp.py

```diff
@@ -20,7 +20,7 @@
 try:
     class OcppMixin:
         def _get_formatted_time(self: OptionalProtocol) -> str:
-            return datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ")
+            return datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Bei der Gelegenheit würde ich gleich von der manuellen Formatierung Abstand nehmen:
```suggestion
            return datetime.now(timezone.utc).isoformat()
```
Alternativ könnte auch die lokale Zeitzone verwendet werden, was das lokale debuggen etwas einfacher machen würde, aber UTC wird ja als "strongly recommended" genannt.
```python
datetime.now().astimezone().isoformat()
```